### PR TITLE
Documentation updates for new keys for PKCS#11 unsealing

### PR DESCRIPTION
**Pull request by stevendpclark**
These new features have made it out into the new Vault releases 1.8.8 and 1.9.3

### CODE REVIEW

File: website/content/docs/configuration/seal/pkcs11.mdx [hunk 1]
```diff
@@ -63,13 +63,13 @@ These parameters apply to the `seal` stanza in the Vault configuration file:
...
-  specified as a string (e.g. `"2305843009213693953"`). May also be specified by 
+  specified as a string (e.g. `"2305843009213693953"`). May also be specified by
   the `VAULT_HSM_SLOT` environment variable.
 
-  ~> **Note**: Slots are typically listed as hex-decimal values in the OS setup 
-  utility and their decimal equivalent is what should be used. For example using the 
-  HSM command-line `pkcs11-tool` a slot listed as `0x2000000000000001`in hex is equal 
-  to `2305843009213693953` in decimal; these values may be listed shorter or 
+  ~> **Note**: Slots are typically listed as hex-decimal values in the OS setup
+  utility and their decimal equivalent is what should be used. For example using the
```
Comment (taoism4504) [inline website/content/docs/configuration/seal/pkcs11.mdx:70]:
```suggestion
  utility and their decimal equivalent is what should be used. For example, using the
```

File: website/content/docs/configuration/seal/pkcs11.mdx [hunk 2]
```diff
@@ -63,13 +63,13 @@ These parameters apply to the `seal` stanza in the Vault configuration file:
...
+  HSM command-line `pkcs11-tool` a slot listed as `0x2000000000000001`in hex is equal
```
Comment (taoism4504) [inline website/content/docs/configuration/seal/pkcs11.mdx:71]:
```suggestion
  HSM command-line `pkcs11-tool`, a slot listed as `0x2000000000000001`in hex is equal
```

**Review by taoism4504 — COMMENTED** [review]
Small nits, but otherwise LGTM.

File: website/content/docs/configuration/seal/pkcs11.mdx [hunk 3]
```diff
@@ -63,13 +63,13 @@ These parameters apply to the `seal` stanza in the Vault configuration file:
...
-  specified as a string (e.g. `"2305843009213693953"`). May also be specified by 
+  specified as a string (e.g. `"2305843009213693953"`). May also be specified by
   the `VAULT_HSM_SLOT` environment variable.
 
-  ~> **Note**: Slots are typically listed as hex-decimal values in the OS setup 
-  utility and their decimal equivalent is what should be used. For example using the 
-  HSM command-line `pkcs11-tool` a slot listed as `0x2000000000000001`in hex is equal 
-  to `2305843009213693953` in decimal; these values may be listed shorter or 
+  ~> **Note**: Slots are typically listed as hex-decimal values in the OS setup
+  utility and their decimal equivalent is what should be used. For example, using the
```
Comment (cipherboy) [inline website/content/docs/configuration/seal/pkcs11.mdx:70]:
```suggestion
  utility but this configuration uses their decimal equivalent. For example, using the
```

Perhaps there's still a cleaner wording I'm not thinking of this morning?

**Review by taoism4504 — APPROVED** [review]
LG

**Pull request merged**